Mathlib/CategoryTheory/FiberedCategory/Cocartesian.lean

```diff
@@ -0,0 +1,329 @@
+/-
+Copyright (c) 2024 Calle Sönne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Calle Sönne
+-/
+
+import Mathlib.CategoryTheory.FiberedCategory.HomLift
+
+/-!
+# Cocartesian morphisms
+
+This file defines cartesian resp. strongly cartesian morphisms with respect to a functor
+`p : 𝒳 ⥤ 𝒮`.
+
+All definitions in this file are opposites of the results in `FiberedCategory/Cartesian`.
```

Comment:
You should add a comment in the other file about keeping these two files in sync.

Reply:
Yes, thanks!